modules/Microsoft.Sql/servers/databases/backupShortTermRetentionPolicies/deploy.bicep

```diff
@@ -0,0 +1,32 @@
+@description('Required. The name of the parent SQL Server.')
+param serverName string
+
+@description('Required. The name of the parent database.')
+param databaseName string
+
+@description('Optional. Differential backup interval in hours.')
+param diffBackupIntervalInHours int = 24
+
+@description('Optional. Poin-in-time retention in days.')
+param retentionDays int = 7
+
+resource server 'Microsoft.Sql/servers@2022-05-01-preview' existing = {
+  name: serverName
+}
+
+resource database 'Microsoft.Sql/servers/databases@2022-05-01-preview' existing = {
+  name: databaseName
+  parent: server
+}
+
+resource backupShortTermRetentionPolicy 'Microsoft.Sql/servers/databases/backupShortTermRetentionPolicies@2022-05-01-preview' = {
+  name: 'default'
```

Comment:
I pressume the name will always be `'default'`, so this proposal is really just for consistency

Reply:
I am not sure if that makes sense to me. The deployment will fail if name is not "default", seems like it would be confusing to have it as an optional parameter then.

Reply:
And I get where you're coming from. I don't even know why it is possible to specify it in the first place - unless the Product Groups want to keep the door open for later changes.

However, for the time being I'd rather like to push for consistency with similar cases like the blobServices, fileServices, etc. modules and would suggest that we tackle this approach for all modules in a dedicated issue. I'll open one for that purpose and like it here asap.

Reply:
Opened issue #2807

Reply:
Closing as per linked issue